model/scratch/dffml-model-anomalydetection/setup.py

```diff
@@ -0,0 +1,16 @@
+import os
+import importlib.util
+from setuptools import setup
+
+# Boilerplate to load commonalities
+spec = importlib.util.spec_from_file_location(
+    "setup_common", os.path.join(os.path.dirname(__file__), "setup_common.py")
+)
+common = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(common)
+
+common.KWARGS["entry_points"] = {
+    "dffml.model": [f"myslr = {common.IMPORT_NAME}.myslr:MySLRModel"]
```

Comment:
Register "anomalydetection" entrypoint. You can check https://github.com/intel/dffml/blob/0b84ca2af45671a0ca3ffce8485a1a2f10b1a7c5/model/scikit/setup.py#L83 for reference.